src/Lecture2.hs

```diff
@@ -178,7 +387,11 @@ False
 True
 -}
 isIncreasing :: [Int] -> Bool
-isIncreasing = error "TODO"
+isIncreasing []  = True
+isIncreasing [_] = True
+isIncreasing (x:y:xs)
+  | x > y     = False
+  | otherwise = isIncreasing (y:xs)
```

Comment:
You can implement `isIncreasing` in a slightly more elegant way by relying on laziness in Haskell 🦥 

```suggestion
isIncreasing (x : y : ys) = x < y && isIncreasing (y : ys)
```

Reply:
updated